app-modules/audit/src/Filament/Resources/AuditResource/Pages/ListAudits.php

```diff
@@ -63,14 +68,30 @@ public function table(Table $table): Table
                 TextColumn::make('auditable_type')
                     ->label('Auditable')
                     ->sortable(),
-                TextColumn::make('user.name')
+                TextColumn::make('change_agent_name')
                     ->label('Change Agent (User)')
-                    ->sortable(),
+                    ->sortable()
+                    ->default('System')
+                    ->visible(Feature::active('change-agent-name')),
```

Comment:
If the `change-agent-name` feature is not yet active, this will result in this column being missing, which would be considered a regression.

We need to one of the following options here:
1. Modify this column to display the name of the `user` relation if `change-agent-name` is NOT active. Like it did before
2. Add `change_agent_name` as its own column that is only visible if `change-agent-name` is active. And leave the `user.name` column but only make it visible if `change-agent-name` is NOT active. We would then remove the `user.name` column when we did a cleanup of the `change-agent-name` flag.